documentation/docs/quick-start.md

```diff
@@ -841,32 +849,41 @@ agent.handleReceivedMessagesEvents().collect { list ->
 </TabItem>
 </Tabs>
 
-### Store the Issued Credential [Holder]
+### Store the Issued Credential **Holder**
 :::caution
 
 The sample application are using an insecure storage solution which should only be used for testing purposes and not production environments!
 
 :::
 
-<summary>Code examples</summary>
+#### Code examples
+
 6. Once the Holder receives a credential from the Cloud Agent, it needs to store the credential somewhere:
 
 <Tabs>
 <TabItem value="js" label="Typescript">
 
-
-
 ```typescript
-props.agent.addListener(ListenerKey.MESSAGE,async (newMessages:SDK.Domain.Message[]) => {
-    //newMessages can contain any didcomm message that is received, including
-    //Credential Offers, Issued credentials and Request Presentation Messages
-    const issuedCredentials = newMessages.filter((message) => message.piuri === "https://didcomm.org/issue-credential/2.0/issue-credential");
-    if (issuedCredentials.length) {
-        for(const issuedCredential of issuedCredentials) {
-            const issueCredential = IssueCredential.fromMessage(issuedCredential);
-            await props.agent.processIssuedCredentialMessage(issueCredential);
-        }
+agent.addListener(SDK.ListenerKey.MESSAGE, async (newMessages: SDK.Domain.Message[]) => {
+  // newMessages can contain any didcomm message that is received, including
+  // Credential Offers, Issued credentials and Request Presentation Messages
+  const issuedCredentials = newMessages.filter(
+    (message) => message.piuri === 'https://didcomm.org/issue-credential/3.0/issue-credential'
```

Comment:
Propose to replace it from
'https://didcomm.org/issue-credential/3.0/issue-credential'
to 
SDK.ProtocolType.DidcommIssueCredential